packages/integrations/lit/server-shim.js

```diff
@@ -3,12 +3,11 @@ import { installWindowOnGlobal } from '@lit-labs/ssr/lib/dom-shim.js';
 if (typeof fetch === 'function') {
 	const _fetch = fetch;
 	installWindowOnGlobal();
-	globalThis.fetch = window.fetch = _fetch;
+	globalThis.fetch = _fetch;
 } else {
 	installWindowOnGlobal();
 }
 
-window.global = window;
 document.getElementsByTagName = () => [];
 // See https://github.com/lit/lit/issues/2393
 document.currentScript = null;
```

Comment:
These also may be unneeded. Not sure about `document.getElementsByTagName` but `document.currentScript` was definitely needed due to the introduction of `window` in global, which will not be needed with the shims in the node builds.

Reply:
`document.gEBTN` definitely needed in server.js